### ISSUE TITLE
Invalid 'main' field

### ISSUE THREAD

**Issue by binarykitchen** [DianaSuvorova/eslint-plugin-react-redux#74]
```
(node:21201) [DEP0128] DeprecationWarning: Invalid 'main' field in '/Users/michael-heuberger/code/test/node_modules/eslint-plugin-react-redux/package.json' of 'dist/index.js'. Please either fix that or report it to the module author
```

**Comment by ljharb** [comment]
How did you run across this warning? I don't see anything deprecated about this package's main field.

**Comment by binarykitchen** [comment]
Only comes up when you use latest node and npm

**Comment by ljharb** [comment]
@binarykitchen can you confirm which exact version of the package you installed, and how you installed it?

**Comment by binarykitchen** [comment]
For some reasons not happening anymore. Dependencies must have changed meanwhile. All good.

**Comment by Snaptags** [comment]
I still have this issue using v3.3.2 with npm v6.14.14.

**Comment by jminty-liverton** [comment]
I'm receiving this warning when building with node 16 stable. This issue should be re-opened.

node v16.13.0
npm 8.1.0, 

"eslint-plugin-react-redux": "3.3.2",

**Comment by jrock2004** [comment]
Can this be re-opened since other ticket requested the user to do it and the ticket here seems to still be closed?
